clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java

```diff
@@ -45,36 +41,6 @@ static void setupTracer() {
     JaegerTestTracer.register();
```

Comment:
I think 
```
@BeforeAll
  static void setupTracer() {
    JaegerTestTracer.register();
  }
```
can be removed now right?

Reply:
Ehm - no - `testTracing` is in this class

Reply:
then I guess we can remove ^ from `TestHttpClientBuilder` :)